tests/unit/Infrastructure/Manage/Client/IdentityProviderClientTest.php

```diff
@@ -38,15 +41,22 @@ class IdentityProviderClientTest extends MockeryTestCase
      */
     private $mockHandler;
 
+    /**
+     * @var Config|Mock
```

Comment:
Technically this would be `Config&Mock`, but that syntax isn't used very widely yet.

Reply:
Cool, I certainly was not aware we can use an ampersand here. The more you know.